fontforge/tottfgpos.c

```diff
@@ -591,14 +616,32 @@ return( calloc(1,sizeof(SplineChar *)) );
 return( glyphs );
 }
 
+static SplineChar **TTFGlyphsFromNames(SplineFont *sf,char *names) {
```

Comment:
And I definitely need to add a comment explaining what this function does. It is a wrapper for SFGlyphsFromNames that forwards only glyphs with valid TrueType encodings. Which, when we're exporting to TrueType, is all we need and all we can do safely.